tests/test_backends.py

```diff
@@ -97,6 +97,24 @@ def test_excited_state_probabilities_circuit(backend):
     np.testing.assert_allclose(probs, target_probs, atol=0.05)
 
 
+@pytest.mark.qpu
+@pytest.mark.xfail(raises=AssertionError, reason="Probabilities are not well calibrated")
+def test_superposition_for_all_qubits(backend):
+    """Applies an H gate to each qubit of the circuit and measures the probabilities."""
+    nqubits = backend.platform.nqubits
+    probs = []
+    for q in range(nqubits):
+        circuit = Circuit(nqubits)
+        circuit.add(gates.H(q=q))
+        circuit.add(gates.M(q=q))
```

Comment:
```suggestion
        circuit.add(gates.M(q))
```
This should fix tests